worker-jobtracking-container/src/test/java/com/hpe/caf/worker/jobtracking/JobTrackingWorkerIT.java

```diff
@@ -98,15 +101,60 @@ public void testTrackingReportTasks() throws Exception
         jobDatabase.createJobTask(defaultPartitionId, jobTaskId, "testProxiedActiveMessage");
         for (int i = 0; i <= 4; i++) {
             final JobStatus status = i < 4 ? JobStatus.Waiting : JobStatus.Completed;
+            final int percentageCompleted = i < 4 ?  0 : 100;
```

Comment:
I'd suggest using an `if` statement to avoid repeating the `i < 4` and help clarify the link between the two variables:

```suggestion
            final JobStatus status;
            final int percentageCompleted;
            if (i < 4) {
                status = JobStatus.Waiting;
                percentageCompleted = 0;
            } else {
                status = JobStatus.Completed;
                percentageCompleted = 100;
            }
```

Note that `status` and `percentageCompleted` are still declared `final` when done like this, so that they can only be initialized once and cannot be used until they have been initialized.